iothub/device/src/AmqpConnectionPoolSettings.cs

```diff
@@ -6,13 +6,12 @@
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// contains Amqp Connection Pool settings for DeviceClient
+    /// contains AMQP Connection Pool settings for DeviceClient
```

Comment:
nit: C caps! :D

Reply:
I will change that in a following PR since I already ran the gates